cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -512,22 +512,19 @@
     public static final String SERVER_ADDRESS = "metrics.query.bind.address";
     public static final String SERVER_PORT = "metrics.query.bind.port";
 
-    public static final String KAFKA_TOPIC_PREFIX = "metrics.kafka.topic.prefix";
```

Comment:
oh, I see your point. The user might have a different setting for this prefix in cdap-site.xml